### ISSUE TITLE
Can't open editor after using Translation remaps

### ISSUE THREAD

**Issue by volzhs** [okamstudio/godot#1729]
I'm using 1.1 beta1.

After added translations and remaps,
I can't open editor.

this is engine.cfg
```
[application]

name="LocalizeTest"
main_scene="res://new_scene.scn"
icon="icon.png"

[locale]

translation_remaps={"res://comic.fnt":["res://comic_bold.fnt:ja"]}
translations=["res://locale.en.xl", "res://locale.es.xl", "res://locale.ja.xl"]
```

and when trying to open editor, console keep saying this.
```
At:  core/io/config_file.cpp:371
ERROR: _get_chunk:  Condition ' level!=0 ' is true. returned: Variant()
```

I can open editor when remove ``translation_remaps={"res://comic.fnt":["res://comic_bold.fnt:ja"]}``

**Comment by reduz** [comment]
i can't reproduce, can you make a broken scene and post it?

**Comment by volzhs** [comment]
https://drive.google.com/file/d/0ByuhCIzeus36cGhaSXBCNnRFU1U/view?usp=sharing

Here is my test project.

**Comment by volzhs** [comment]
Did I do something wrong way to add remaps for locale?

This is what I did with 1.0 stable, and 1.1 beta1.

1. make new project
2. make a scene with Node2D
3. save it and set main_scene in Project settings.
4. import a 2d texture(test.tex)
5. add test.tex at Project settins > Localization > Remaps.
6. select test.tex resource and add same test.tex for English locale at Remaps by Locale.
7. save it and Quit to Project List.

then this error occured with 1.0 stable and 1.1beta1.
Did I add remaps with wrong way?

**Comment by volzhs** [comment]
I want this issue will be fixed before 1.1 stable released.

**Comment by volzhs** [comment]
is there any working example project which used Localizing Resources(Localization -> remaps)?

**Comment by reduz** [comment]
I can't manage to break the editor.. but I will make a demo I guess

**Comment by volzhs** [comment]
Thanks for the demo.
I saw your demo, but it doesn't contains resource ``remaps``.

Translation works fine.
But what I need is using resource remaps.

![image](https://cloud.githubusercontent.com/assets/8281454/7581119/dd67c336-f8b9-11e4-90bf-54518900ed5f.png)

I want add resource remaps here.
so engine.cfg contains ``translation_remaps={"res://comic.fnt":["res://comic_bold.fnt:ja"]}`` when I add some file.

this part causes problem.

**Comment by reduz** [comment]
I think I fixed this issue, and also added a demo in gui/translation which includes translation remaps,
please test again and let me know if it works.

**Comment by volzhs** [comment]
I pulled latest source and compiled windows tool successfully by ``scons -j 4 p=windows target=release_debug bits=32`` on linux. (I can't compile it on windows)
And copied ``godot.windows.opt.tools.32.exe`` to windows and ran it.
After that, I imported your ``demos/gui/translation``.
Then, console keeps saying this.
```
ERROR: _get_chunk:  Condition ' level!=0 ' is true. returned: Variant()
At:  core/io/config_file.cpp:371
```

I tested it on linux too. (godot.x11.opt.tools.64)
The result was same.

It's same error message as before.
